lib/modules/apostrophe-users/index.js

```diff
@@ -26,13 +26,20 @@
 // In our experience, combining the concept of a "user" who can log in and do things
 // with the concept of a "staff member" who appears in a staff directory is more
 // trouble than it is worth. That's why the `published` field is not present in
-// `apostrophe-users`. You can add it back in, but then you have to deal with
-// the confusing concept of "users" who shouldn't actually be allowed to log in.
+// `apostrophe-users`. You can add it back in, but **then you have to deal with
+// the potentially confusing concept of "users" who shouldn't actually be
+// allowed to log in**.
 //
 // So for a staff directory, we suggest you create a separate `employee` module
 // or similar, extending `apostrophe-pieces`, unless it's true that basically
 // everyone should be allowed to log in.
 //
+// ::: tip
+// To bring the `published` field back for the `apostrophe-users` piece type,
+// set the `defaultRemoveFields`, `defaultRemoveFilters`, and
+// `defaultRemoveColumns` properties on that module to an empty array.
```

Comment:
Have you verified that you can still log in and perform other user actions like editing as a user who is not "published" once that field has been brought back? Note that when you make this transition all users will be unpublished as of that moment.